docs/api/fleet/agent_api_checkin.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[fleet-agent-checkin]]
+=== Fleet agent checkin API
+++++
+<titleabbrev>Agent checkin</titleabbrev>
+++++
+
+This API is used by a fleet agent to report is current state.
+
+[[fleet-agent-checkin-request]]
+==== Request
+
+`POST /api/fleet/agents/{agentId}/checkin` 
+
+==== Headers
+
+`kbn-fleet-access-token`::
+  (Required, string) A fleet agent access token.
+
+[[fleet-agent-checkin-request-body]]
+==== Request body
+
+`events`::
+  (Required, array) An array of events with the properties `type`, `subtype`, `message`, `timestamp`, `payload`, and `data`.
+
+`local_metadata`::
+  (Optional, object) an object that contains the local metadata for an agent. Metadata are a dictonary of strings (example: `{ "os": "macos" }`).
+
+[[fleet-agent-checkin-request-request-codes]]
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+[[fleet-agent-checkin-example]]
+==== Example
+
+[source,js]
+--------------------------------------------------
+POST /api/fleet/agents/a4937110-e53e-11e9-934f-47a8e38a522c/checkin
```

Comment:
Not sure the indentation in the output is what you are expecting (maybe you're using tabs and should use spaces instead)?

![image](https://user-images.githubusercontent.com/14206422/66243696-6687eb80-e6ba-11e9-91bc-a7d55e91c062.png)

Reply:
Good catch my IDE it's not handling asciidoc really well